mtbl.pyx

```diff
@@ -612,7 +655,7 @@ cdef class sorter(object):
                                            <mtbl_merge_func> merge_func_wrapper,
                                            <void *> merge_func)
 
-        mtbl_sorter_options_set_temp_dir(opt, temp_dir)
+        mtbl_sorter_options_set_temp_dir(opt, temp_dir.encode('UTF-8'))
```

Comment:
Inconsistent. In other places we use only `encode()`. I think that this is probably the way that we would want to do it, or maybe check to see if the string has an encoding set already and default to utf-8 if it doesn't.

Reply:
utf-8 is already the default for py3, but fixed by just specifying 'utf-8' for all those calls for encoding filenames: https://github.com/farsightsec/pymtbl/pull/10/commits/d848723bdb37d91ad0c54066092971eb877a5ff3